tests/__init__.py

```
@@ -155,63 +138,225 @@
-	test_hear_data_processing.RecvDataProcessingTestSuite,
-	test_hear_data_processing.HearHandleNoneDataTestSuite,
-	test_hear_cleanup.HearCleanupTestSuite,
-	test_hear_keyboard_interrupt.TestHearKeyboardInterrupt
-)
-
-
-def load_tests(loader, tests, pattern):
-	"""Will Load the tests from the project and then attempts to load the doctests too.
 
-	Testing:
+def loadDocstringsFromModule(module) -> unittest.TestSuite:
```

Ruff (0.8.2)

Missing type annotation for function argument `module`

(ANN001)